skycultures/chinese_chenzhuo/index.json

```diff
@@ -0,0 +1,1856 @@
+    "___:___ __ 12:43:34 +80:33:39 18:05:48 -30:25:27 07A 08A",
+    "___:___ __ 18:05:48 -30:25:27 00:43:34 -80:33:39 07A 08A",
+    "___:___ __ 12:43:34 +80:33:39 18:45:39 -26:59:27 08A 09A",
+    "___:___ __ 18:45:39 -26:59:27 00:43:34 -80:33:39 08A 09A",
+    "___:___ __ 12:43:34 +80:33:39 20:21:01 -14:46:53 09A 10A",
+    "___:___ __ 20:21:01 -14:46:53 00:43:34 -80:33:39 09A 10A",
+    "___:___ __ 12:43:34 +80:33:39 20:47:41 -09:29:45 10A 11A",
+    "___:___ __ 20:47:41 -09:29:45 00:43:34 -80:33:39 10A 11A",
+    "___:___ __ 12:43:34 +80:33:39 21:31:34 -05:34:16 11A 12A",
+    "___:___ __ 21:31:34 -05:34:16 00:43:34 -80:33:39 11A 12A",
+    "___:___ __ 12:43:34 +80:33:39 22:05:47 -00:19:11 12A 13A",
+    "___:___ __ 22:05:47 -00:19:11 00:43:34 -80:33:39 12A 13A",
+    "___:___ __ 12:43:34 +80:33:39 23:04:46 +15:12:19 13A 14A",
+    "___:___ __ 23:04:46 +15:12:19 00:43:34 -80:33:39 13A 14A",
+    "___:___ __ 12:43:34 +80:33:39 00:13:14 +15:11:01 14A 15A",
+    "___:___ __ 00:13:14 +15:11:01 00:43:34 -80:33:39 14A 15A",
+    "___:___ __ 12:43:34 +80:33:39 00:47:20 +24:16:02 15A 16A",
+    "___:___ __ 00:47:20 +24:16:02 00:43:34 -80:33:39 15A 16A",
+    "___:___ __ 12:43:34 +80:33:39 01:54:38 +20:48:29 16A 17A",
+    "___:___ __ 01:54:38 +20:48:29 00:43:34 -80:33:39 16A 17A",
+    "___:___ __ 12:43:34 +80:33:39 02:43:27 +27:42:26 17A 18A",
+    "___:___ __ 02:43:27 +27:42:26 00:43:34 -80:33:39 17A 18A",
+    "___:___ __ 12:43:34 +80:33:39 03:44:53 +24:06:48 18A 19A",
+    "___:___ __ 03:44:53 +24:06:48 00:43:34 -80:33:39 18A 19A",
+    "___:___ __ 12:43:34 +80:33:39 04:28:37 +19:10:50 19A 20A",
+    "___:___ __ 04:28:37 +19:10:50 00:43:34 -80:33:39 19A 20A",
+    "___:___ __ 12:43:34 +80:33:39 05:34:49 +09:29:22 20A 21A",
+    "___:___ __ 05:34:49 +09:29:22 00:43:34 -80:33:39 20A 21A",
+    "___:___ __ 12:43:34 +80:33:39 05:32:00 -00:17:57 21A 22A",
+    "___:___ __ 05:32:00 -00:17:57 00:43:34 -80:33:39 21A 22A",
+    "___:___ __ 12:43:34 +80:33:39 06:22:58 +22:30:49 22A 23A",
+    "___:___ __ 06:22:58 +22:30:49 00:43:34 -80:33:39 22A 23A",
+    "___:___ __ 12:43:34 +80:33:39 08:31:36 +18:05:40 23A 24A",
+    "___:___ __ 08:31:36 +18:05:40 00:43:34 -80:33:39 23A 24A",
+    "___:___ __ 12:43:34 +80:33:39 08:37:39 +05:42:14 24A 25A",
+    "___:___ __ 08:37:39 +05:42:14 00:43:34 -80:33:39 24A 25A",
+    "___:___ __ 12:43:34 +80:33:39 09:27:35 -08:39:31 25A 26A",
+    "___:___ __ 09:27:35 -08:39:31 00:43:34 -80:33:39 25A 26A",
+    "___:___ __ 12:43:34 +80:33:39 09:51:29 -14:50:48 26A 27A",
+    "___:___ __ 09:51:29 -14:50:48 00:43:34 -80:33:39 26A 27A",
+    "___:___ __ 12:43:34 +80:33:39 10:59:46 -18:17:56 27A 28A",
+    "___:___ __ 10:59:46 -18:17:56 00:43:34 -80:33:39 27A 28A",
+    "___:___ __ 12:43:34 +80:33:39 12:15:48 -17:32:31 28A 01A",
+    "___:___ __ 12:15:48 -17:32:31 00:43:34 -80:33:39 28A 01A"
+  ],
+  "common_names": {
+    "HIP 746": [{"english": "Wangiang/Wang Liang", "native": "王良"}],
```

Comment:
The contents of "english" should be split into "english" and "pronounce".

Reply:
Actually, it seems the first word is misspelled (is missing an "L").

Reply:
interesting... yes, the "L" Is missing but well, both spellings are possible "Wangliang" and "Wang Liang" - AFAIK, that's a (name of a general - eponymous).

Reply:
we can use it for both, "pronounce" and "english"

Reply:
If you look at the whole series, one is always english, the other is always pronunciation. If anyone finds time, the list should be split, at the latest in 25.2.